bee-node/build_and_run.sh

```diff
@@ -0,0 +1,78 @@
+# docker image tag from Cargo.toml in this way the version will be in sync
+IMAGE_TAG=`cat ./Cargo.toml | grep version | head -1 | sed 's/["version=]//g' | tr -d '[[:space:]]'`
+export IMAGE_TAG=${IMAGE_TAG}
+BUILD_BEE="cargo build --release"
+BUILD_BEE_WITH_DASHBOARD="cargo build --release --features dashboard"
+DOCKER_BUILD_BEE="docker build -t bee:${IMAGE_TAG} -f docker/Dockerfile ../"
+RUN_DOCKER_COMPOSE="docker-compose -f ./docker/bee.yaml up"
+
+# build bee node
+function build_bee {
+    ${BUILD_BEE}
+}
+# build bee node with dashboard feature
```

Comment:
Newline here.

Reply:
Hi sorry for the delay, I just inserted new lines